packages/documentation/copy/en/developer/Getting_Started_Linux.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Getting started on Linux 
+layout: docs
+permalink: /docs/handbook/download
+oneline: "Quick start with Lingua Franca on Linux"
+preamble: >
+---
+
+# Getting started on Linux
+
+## Installation
+
+### Debian & Other
+
+Sadly there are currently no packaged version of the lingua-franca compiler in debian distributions so there is
+really only the manual way. But we offer prebuild binaries [here](https://github.com/lf-lang/lingua-franca/releases). 
+Please note that you need a modern version of java 11 or greater.
+
+If you want to compile the lingua-franca compiler from scratch there is a [guide](Downloading\ and\ Building.md) for that. 
+
+```
```

Comment:
What is below is downloading `lfc`, which seems to have little to do with the sentence above.